src/main/java/racingcar/domain/UserConsole.java

```diff
@@ -0,0 +1,40 @@
+package racingcar.domain;
+
+import camp.nextstep.edu.missionutils.Console;
+import racingcar.domain.Car;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class UserConsole {
+    public ArrayList<Car> makingCarList(){
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String carName = Console.readLine();
+        if(carName.equals("")){
+            throw new IllegalArgumentException();
+        }
+        ArrayList<String> carNames = new ArrayList<String>(Arrays.asList(carName.split(",")));
+        ArrayList<Car> carList = new ArrayList<>();
+        if(carNames.size() < 1){
+            throw new IllegalArgumentException();
+        }
+        for(int i=0; i<carNames.size(); i++){
+            if(carNames.get(i).length() > 5){
```

Comment:
```suggestion
            if(carNames.get(i).length() > MAX_NAME_LEGNTH){
```

매직넘버는 코드가 방대해질수록 수정지점을 늘리고, 가독성을 저해합니다. 웬만하면 상수로 치환하여 사용해주세요!